PD-1862 / 25.10 / Pd 1862 create enterprise ad content (#3678)
truenas/documentation: content/SCALE/SCALEUIReference/Credentials/LocalGroupsScreens.md

{{< trueimage src="/images/SCALE/Credentials/GroupsListedSCALE.png" alt="Groups Screen" id="Groups Screen" >}}

> When enabled, the **Show Built-In Groups** toggl turns bluee and shows built-in groups. When disabled, the toggle turns grey and shows only non-built-in groups.

Reviewer: ```suggestion
When enabled, the **Show Built-In Groups** toggle turns blue and shows built-in groups. When disabled, the toggle turns grey and shows only non-built-in groups.
```

> When enabled, the **Show Built-In Groups** toggl turns bluee and shows built-in groups. When disabled, the toggle turns grey and shows only non-built-in groups.

Reviewer: gray instead of grey perhaps?

Author: gray is the US spelling of the color where grey is the UK version. Both are correct. I default to spelling grey. ;-)

| **Name** | (Required) Assigns the entered name to the group. A group name cannot begin with a hyphen (-) or contain a space, tab, or any of these characters: colon (:), plus (+), ampersand (&), hash (#), percent (%), carat (^), open or close parentheses ( ), exclamation mark (!), at symbol (@), tilde (~), asterisk (*), question mark (?) greater or less than (<) (>), equal (=). You can only use the dollar sign ($) as the last character in a group name or username. |
| **Privileges** | Attaches a role privilege to the group as assigned and configured on the Add or **Edit Privileges** screens. <br>Using custom administrator roles aside from the defaults is an experimental feature and is not supported.<br>Do not modify the local administrator or default admin user privileges! Only use if you need users in this group to access limited areas of the TrueNAS UI or authentication for TrueNAS API calls. |

> | **Allowed sudo commands** | Permits the group members to enter the specific [sudo](https://www.sudo.ws/) commands enter in this field. Enter each command as an absolute path to the ELF (Executable and Linkable Format) executable file, for example */usr/bin/nano*. <file>/usr/bin/</file> is the default location for commands. <br> Grants limited root-like permissions for group members when using these commands. Using sudo prompts the user for their account password. |

Reviewer: ```suggestion
| **Allowed sudo commands** | Permits the group members to enter the specific [sudo](https://www.sudo.ws/) commands in this field. Enter each command as an absolute path to the ELF (Executable and Linkable Format) executable file, for example */usr/bin/nano*. <file>/usr/bin/</file> is the default location for commands. <br> Grants limited root-like permissions for group members when using these commands. Using sudo prompts the user for their account password. |
```

## Privileges Screen

{{< hint type="warning" title="Experimental Feature" >}}
The **Privileges** feature in an early releases was an experimental feature but is no longer experimental as of 24.10.

> Do not edit the existing predefined administrator roles (Full Control Admin, Rreadonly Admin, and Sharing Admin)!

Reviewer: ```suggestion
Do not edit the existing predefined administrator roles (Full Control Admin, Readonly Admin, and Sharing Admin)!
```
